canonical/snapcraft.io
# Update discourse

## Done
- Update discourse version
## How to QA
- Go to docs/installing-snap-on-manjaro-linux and ensure that the image on the page displays.
## Issue / Card https://warthogs.atlassian.net/browse/WD-7920
Fixes #

## Screenshots
